client/checkout/classic/upe-split.js

```diff
@@ -701,14 +702,11 @@ jQuery( function ( $ ) {
 	/**
 	 * Checks if the customer is using a saved payment method.
 	 *
-	 * @param {string} paymentMethodType Stripe payment method type ID.
 	 * @return {boolean} Boolean indicating whether or not a saved payment method is being used.
 	 */
-	function isUsingSavedPaymentMethod( paymentMethodType ) {
```

Comment:
This will make SEPA fail once attempting to make it compatible with Split UPE, but patching this way to make things work